src/codex_plus/request_logger.py

```
@@ -0,0 +1,91 @@
+    async def _log_payload_to_file_async(body: bytes) -> None:
+        """Log payload to branch-specific directory asynchronously"""
+        # Get current git branch name asynchronously
+        try:
+            proc = await asyncio.create_subprocess_exec(
+                "git", "rev-parse", "--abbrev-ref", "HEAD",
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.DEVNULL
+            )
+            stdout, _ = await asyncio.wait_for(proc.communicate(), timeout=3.0)
+            branch = stdout.decode().strip() if stdout else "unknown"
+        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)